`azurerm_network_manager` - `scope_accesses` is now optional (#28781)
hashicorp/terraform-provider-azurerm: website/docs/r/network_manager.html.markdown

The following arguments are supported:

* `name` - (Required) Specifies the name which should be used for this Network Manager. Changing this forces a new Network Manager to be created.

* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Network Manager should exist. Changing this forces a new Network Manager to be created.

* `location` - (Required) Specifies the Azure Region where the Network Manager should exist. Changing this forces a new resource to be created.

* `scope` - (Required) A `scope` block as defined below.

* `description` - (Optional) A description of the Network Manager.

> * `scope_accesses` - (Optional) A list of configuration deployment type. Possible values are `Connectivity`, `SecurityAdmin` and `Routing`, corresponds to if Connectivity Configuration, Security Admin Configuration or Routing Configuration is allowed for the Network Manager.

Reviewer: ```suggestion
* `scope_accesses` - (Optional) A list of configuration deployment types. Possible values are `Connectivity`, `SecurityAdmin` and `Routing`, which specify whether Connectivity Configuration, Security Admin Configuration or Routing Configuration are allowed for the Network Manager.
```

Author: fixed
